docs/maps/vector-style.asciidoc

```diff
@@ -10,12 +12,57 @@
 
 *Symbol size*:: Defines the symbol size of point features.
 
-Click the *link* button to toggle between static styling and data-driven styling.
+NOTE: *LineString* and *MultiLineString* geometries do not have fill and do not use fill color property.
+Set border color to style line geometries
+
+[float]
+[[maps-vector-style-static]]
+==== Static styling
+
+Use static styling to specificy a constant value for a style property.
 
 [role="screenshot"]
 image::maps/images/vector_style_static.png[]
 
+[float]
+[[maps-vector-style-data-driven]]
+==== Data driven styling
+
+Use data driven styling to symbolize features from numeric property values.
+Click image:maps/images/gs_link_icon.png[] to enable data driven styling for a style property.
+
+NOTE: The image:maps/images/gs_link_icon.png[] button is only available when your vector features contain numeric properties.
+
+The image below shows an example using the Kibana sample data set "Sample web logs".
+The *kibana_sample_data_logs* layer uses data driven styling for fill color and symbol size style properties.
+See <<add-sample-data, adding sample data>> to install the `kibana_sample_data_logs` index.
+
+The layer sets fill to a color ramp from light green to dark green.
+The layer specifies that the `hour_of_day` feature property is used to determine the color of each feature based on where the value fits on a linear scale between the minimum and maximum values.
+Light green circles symbolize documents that occur earlier in the day while dark green circles symbolize documents that occur later in the day.
+
+The layer sets symbol size to a range between 1 and 24 pixels.
+The layer specifies that the `bytes` feature property is used to determine the symbol size of each feature based on where the value fits on a linear scale between the minimum and maximum values.
+Smaller circles symbolize documents with smaller payloads while larger circles symbolize documents with larger payloads.
+
 [role="screenshot"]
 image::maps/images/vector_style_dynamic.png[]
 
-NOTE: The *link* button is only available when your vector features contain numeric properties.
+
+[float]
+[[maps-vector-style-classification]]
+==== Classification styling
```

Comment:
Suggestions:

- Tighten the section about sample date
- Use a bulleted list for descriptions about what's going on with the properties

<img width="783" alt="Screen Shot 2019-04-15 at 1 15 04 PM" src="https://user-images.githubusercontent.com/33642766/56162739-98c55580-5f81-11e9-9da2-16069f5ef427.png">